lib/lotus/model/adapters/sql/sqlite_console.rb

```diff
@@ -0,0 +1,28 @@
+module Lotus
+  module Model
+    module Adapters
+      module Sql
+        class SqliteConsole
+          def initialize(uri, options = {})
+            @uri = uri
+            @options = options
+          end
+
+          def connection_string
+            "sqlite3 #{host}#{database}"
+          end
+
+          private
+
+          def host
+            @options.fetch('host') { @uri.host }
```

Comment:
It's worth to parse it through `ShellWord` because URI tend to be full file path